src/main/scala/model/world/Reproduction.scala

```diff
@@ -24,21 +25,34 @@ object Reproduction {
   /**
    * @param mom a bunny
    * @param dad another bunny
-   * @return the 4 children of the couple, one for each cell of the Punnett's square
+   * @return the 4 children of the couple, one for each cell of the Punnet's square
```

Comment:
Punnett* :)

Reply:
scusate lo ricambio a mano che intellij mi corregge l'inglese in automatico ahaha